docs/reference/setup/sysconfig/file-descriptors.asciidoc

```diff
@@ -16,6 +16,9 @@ For the `.zip` and `.tar.gz` packages, set <<ulimit,`ulimit -n 65536`>> as
 root before starting Elasticsearch,   or set `nofile` to `65536` in
 <<limits.conf,`/etc/security/limits.conf`>>.
 
+On macOS, you must also pass the JVM option <<-XX:-MaxFDLimit>>
```

Comment:
The `<<>>` syntax is for referring to other sections in the docs. Here we can use backticks ` surrounding `-XX:-MaxFDLimit` to get the formatting that I think you're looking for.